src/constructors.jl

```diff
@@ -254,7 +254,7 @@ function BitVector(obj::GapObj)
     !Wrappers.IsBlist(obj) && throw(ConversionError(obj, BitVector))
     # TODO: a much better conversion would be possible, at least if `obj` is
     # in IsBlistRep, then we could essentially memcpy data
-    len = Wrappers.Length(obj)
+    len = Wrappers.Length(obj)::Int
```

Comment:
Ah, I have learned something:
`Wrappers.Length` defines its return type to be just `GapInt` (for example because there are infinite lists), but here we know more.
Thus it is worth to think about type annotations also for functions defined in `Wrappers`.